AdblockPlusSafari/AdblockPlusContainerController.m

```diff
@@ -53,6 +53,18 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
   if ([segue.destinationViewController respondsToSelector:@selector(setAdblockPlus:)]) {
     [(id)segue.destinationViewController setAdblockPlus:self.adblockPlus];
   }
+
+#ifdef CONFIGURABLE_CUSTOM_FILTER_LIST_ENABLED
+  if ([segue.identifier isEqualToString:@"ConfigureFilterListsSegue"]) {
```

Comment:
What do you think, is it somewhat normal for iOS apps to use custom labels for the back "button"?